### ISSUE TITLE
Unit handling: Create links between the unit and the respective value

### ISSUE THREAD

**Issue by StephanM87** [StephanM87/Strenda-biocatalysis#18]

**Comment by PeterHalling** [comment]
When I was working with computer presented quizzes some 10 years ago, I was told that open source code to handle values and SI units was close to being finished. Hopefully something is now available that allows us to create a compound type which holds together the value and the unit. (It would also help greatly with searches of data and metadata, so the system can identify that a rate originally entered in mmol h-1 g-1 is exactly in the search range I entered in umol min-1 mg-1.)

**Comment by StephanM87** [comment]
Yes, in principlat this should work. We need to discuss this with Jan, since we are using his sdRDM as the basis to realise the interconnection between the markdow and the Python classes we need. Therefore, I would add this issue to the list of discussion on the next meeting

**Comment by PeterHalling** [comment]
Yes, I agree - but probably the details of how it is done would be a topic for a meeting of a smaller group. I did a quick Google search on "python si units type" and there are clearly various projects and tools around - not sure if they are any good!

**Comment by StephanM87** [comment]
- Reason:

- Helps with the definition of the attributes
- If its not technical possible:
- - Suggest the units which are required
- - Do we propose only 1 unit or do we provide a list of possible units ❗
